backend/game/turn_actions/heal.py

```diff
@@ -0,0 +1,11 @@
+from game.player.player import Player
+import game.values.locations as locations
+
+
+def heal_self_from_dice(player_being_healed: Player, health_points_to_add):
+    if player_being_healed.location is not locations.Locations.TOKYO:
```

Comment:
For readability, I tend to avoid using not whenever possible (just an extra evaluation I have to do in my head).  This can just be changed to equals locations.Locations.OUTSIDE

Also may want to verify that `is` is the correct operator here because could make a case for ==

Reply:
I agree. Also when we add Tokyo Bay, the logic becomes much easier to check outside, vs checking both Tokyos

Reply:
Ooh I completely missed the difference between is and == thanks. Will update.